Objects/setobject.c

```diff
@@ -2233,6 +2241,53 @@ PyTypeObject PyFrozenSet_Type = {
 };
 
 
+PyTypeObject _PyReproducibleFrozenSet_Type = {
```

Comment:
For new types, please use C99 named fields like https://github.com/python/cpython/blob/bb3e0c240bc60fe08d332ff5955d54197f79751c/Doc/includes/custom2.c#L98-L110